person/forms.py

```diff
@@ -210,6 +210,64 @@ def __init__(self, *args, **kwargs):
                 self.fields[field].widget.attrs.update(
                     {'class': 'form-control'})
 
+class FilterChildForm(forms.Form):
+    email = forms.CharField(max_length=100, required=False , label="Búsqueda por email")
+    name = forms.CharField(max_length=50, required=False , label="Búsqueda por nombre")
+    surname = forms.CharField(max_length=50, required=False , label="Búsqueda por apellido")
+    birth_date_min = forms.DateField(required=False, widget=forms.DateInput(attrs={'type': 'date'}), label="Nacido/a después del")
+    birth_date_max = forms.DateField(required=False, widget=forms.DateInput(attrs={'type': 'date'}), label="Nacido/a antes del")
+    sex = forms.ChoiceField(choices=[('', '--Seleccione--'), ('F', 'Femenino'), ('M', 'Masculino'), ('O', 'Otro')], required=False, label="Género")
+    city = forms.CharField(max_length=100, required=False, label="Búsqueda por ciudad")
+    address = forms.CharField(max_length=100, required=False, label="Búsqueda por dirección")
+    telephone = forms.IntegerField(required=False, label="Búsqueda por teléfono", widget=forms.NumberInput(attrs={'min': 0}))
```

Comment:
telephone and postal code should be charfields too. The atrbutes from the model were changed to charfield in order to add prefix like "+34" in numbers, o letters in some postal codes.

Reply:
You are absolutely right Juan, I did not take those cases into account, thank you very much for the indication.